reports/graphs/feeding_amounts.py

```diff
@@ -14,23 +14,57 @@ def feeding_amounts(instances):
     :param instances: a QuerySet of Feeding instances.
     :returns: a tuple of the the graph's html and javascript.
     """
-    totals = {}
+    feeding_types = [
```

Comment:
You can use model metadata to get these types directly from the model (after its imported). Some variation of `feeding_types = Feeding._meta.get_field('type').choices` should do the trick.